src/test/scala/edu/berkeley/ce/sparkrocks/BlockSpec.scala

```diff
@@ -394,7 +394,6 @@ class BlockSpec extends FunSuite {
     val block = Block(Array(1.0, 1.0, 1.0), List(face1, face2, face3))
     val expectedIntersection = Map(
       face1 -> List.empty[Array[Double]],
-      face2 -> List.empty[Array[Double]],
```

Comment:
This was due to the extra `val` that you'd mentioned earlier. `face2` will be removed if you call `nonRedundantFaces`. I added it back in since the extra `val` that triggered a call to `nonRedundantFaces` is now gone.